libs/domain/search/box/src/controllers/query-controls.controller.ts

```diff
@@ -30,7 +31,7 @@ export class QueryControlsController {
         </button>
       </oryx-button>
 
-      <oryx-icon-button slot="suffix" size="small">
+      <oryx-icon-button slot="suffix" size=${Size.Lg}>
```

Comment:
is it correct that we changes `small` into Size.Lg?

Reply:
no, mistake